evc-sumo/Dockerfile

```diff
@@ -0,0 +1,42 @@
+#  Copyright (C) 2023 LEIDOS.
```

Comment:
Curious on your decision to make this a separate image from the rest of the carma-simulation image. Is there a reason to prefer this over just making it an optional part of the carma-simulation image build? I'd think you could make this all a conditional part of the build for the same carma-simulation image. Have the user pass in a token as an argument to `./build-image.sh` which gets propagated as an environment variable (see my comment below) into install.sh which simply checks:

```
if token is not empty:
    do evc pull and install
```

and doesn't attempt to install the evc otherwise.

I ask, because I'm not sure how this image would get run. MOSAIC inside of the carma-simulation image is responsible for starting SUMO so I'm concerned about how this would work in coordination with that if they're in separate images.